### PR TITLE
Safer `Core Bluetooth` presence check

### PR DESCRIPTION
**Pull request by pvieito**
The previous check does not working on macOS Big Sur as it presumes that Core Bluetooth was already loaded on the process.

Superseeds #278

### CODE REVIEW

File: bleak/__init__.py [hunk 1]
```diff
@@ -51,9 +51,9 @@
         BleakClientBlueZDBus as BleakClient,
     )  # noqa
 elif platform.system() == "Darwin":
-    from Foundation import NSClassFromString
-
-    if NSClassFromString("CBPeripheral") is None:
+    try:
+        from CoreBluetooth import CBPeripheral
+    except:
```
Comment (dlech) [inline bleak/__init__.py:56]: How about this instead?

```python
except Exception as ex:
    raise BleakError("Bleak requires the CoreBluetooth Framework") from ex
```
Then we don't lose information from the first exception.

Comment (pvieito) [reply bleak/__init__.py]: 👍🏻

Comment (hbldh) [reply bleak/__init__.py]: You have got to be kidding me! I have been coding Python for 12 years now and this is the first time I have seen or heard of `raise x from y`!

🤯

I have many times lamented Python's lack of good exception chaining and it has been there all along...

@dlech thank you for this revelation!

Comment (dlech) [reply bleak/__init__.py]: :grin:

**Pull request merged**